core/src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -503,11 +503,11 @@ class KotlinCompilation : AbstractKotlinCompilation<K2JVMCompilerArguments>() {
 		if(javaSources.isEmpty())
 			return ExitCode.OK
 
-        if(jdkHome != null) {
+        if(jdkHome != null && jdkHome!!.canonicalPath != processJdkHome.canonicalPath) {
```

Comment:
Would be nice to have a comment here explaining that we do this check to improve the performance.

Reply:
done.